Description: arm and mips targets are cross compiled by default, disable this.
Forwarded: not-needed, upstream do not promote building on same host as target arch.
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2011-12-28
--- a/Makefile
+++ b/Makefile
@@ -216,14 +216,14 @@
 	              -Ibuild/standalone.gypi --depth=. -Dtarget_arch=x64 \
 	              -S-x64 $(GYPFLAGS)
 
-$(OUTDIR)/Makefile-arm: $(GYPFILES) $(ENVFILE) build/armu.gypi
+$(OUTDIR)/Makefile-arm: $(GYPFILES) $(ENVFILE)
 	gyp --generator-output="$(OUTDIR)" build/all.gyp \
-	              -Ibuild/standalone.gypi --depth=. -Ibuild/armu.gypi \
+	              -Ibuild/standalone.gypi --depth=. -Dtarget_arch=arm \
 	              -S-arm $(GYPFLAGS)
 
-$(OUTDIR)/Makefile-mips: $(GYPFILES) $(ENVFILE) build/mipsu.gypi
+$(OUTDIR)/Makefile-mips: $(GYPFILES) $(ENVFILE)
 	gyp --generator-output="$(OUTDIR)" build/all.gyp \
-	              -Ibuild/standalone.gypi --depth=. -Ibuild/mipsu.gypi \
+	              -Ibuild/standalone.gypi --depth=. -Dtarget_arch=mips \
 	              -S-mips $(GYPFLAGS)
 
 $(OUTDIR)/Makefile-native: $(GYPFILES) $(ENVFILE)
